website/docs/cloud-docs/api-docs/private-registry/gpg-keys.mdx

```diff
@@ -43,6 +43,103 @@ These endpoints are only relevant to private providers. When you [publish a priv
 
 You need [owners team](/cloud-docs/users-teams-organizations/permissions#organization-owners) or [Manage Private Registry](/cloud-docs/users-teams-organizations/permissions#manage-private-registry) permissions to add, update, or delete GPG keys in a private registry.
 
+## List GPG Keys
+
+`GET /api/registry/:registry_name/v2/gpg-keys`
+
+### Parameters
+
+| Parameter        | Description        |
+|------------------|--------------------|
+| `:registry_name` | Must be `private`. |
+
+### Query Parameters
+
+This endpoint supports pagination [with standard URL query parameters](/cloud-docs/api-docs/#query-parameters); remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
+
+| Parameter           | Description                                                                                                          |
+|---------------------|----------------------------------------------------------------------------------------------------------------------|
+| `filter[namespace]` | **Required.** A comma-seperated list of one or more namespaces. These must be authorized TFC/TFE organization names. |
+| `page[number]`      | **Optional.** If omitted, the endpoint will return the first page.                                                   |
+| `page[size]`        | **Optional.** If omitted, the endpoint will return 20 GPG keys per page.                                             |
```

Comment:
We're trying to move away from future-facing language in docs. So when we talk about what the product does or is doing, we want to say "Terraform Cloud starts the run" rather than "Terraform Cloud will start the run." A good pro-tip for this is if you're using will....you can usually take it out!

```suggestion
| `page[number]`      | **Optional.** If omitted, the endpoint returns the first page.                                                   |
| `page[size]`        | **Optional.** If omitted, the endpoint returns 20 GPG keys per page.                                             |
```